cnd/src/bitcoin.rs

```diff
@@ -67,10 +81,45 @@ impl<'de> Deserialize<'de> for PublicKey {
             where
                 E: de::Error,
             {
-                v.parse().map(PublicKey::new).map_err(E::custom)
+                v.parse().map_err(E::custom)
             }
         }
 
         deserializer.deserialize_str(PublicKeyVisitor)
     }
 }
+
+#[cfg(test)]
+mod tests {
+    use super::*;
+    use std::str::FromStr;
+
+    #[test]
+    fn serialize_bitcoin_identity() {
+        let secp_pubkey = secp256k1::PublicKey::from_str(
+            "02c2a8efce029526d364c2cf39d89e3cdda05e5df7b2cbfc098b4e3d02b70b5275",
```

Comment:
Perhaps a const instead of these identical strings.

Reply:
Well some are in JSON format and some are not. Which means it's just duplicated once. Let's refactor on the 2rd occurrence :)